Dockerfile

```diff
@@ -6,14 +6,17 @@ LABEL maintainer="Dick Snel <dick.snel@ictu.nl>"
 USER root
 
 RUN mkdir /zap/wrk \
+    && mkdir -p /home/zap/.cache/pip \
 	&& cd /opt \
-	&& wget -qO- -O geckodriver.tar.gz https://github.com/mozilla/geckodriver/releases/download/v0.27.0/geckodriver-v0.27.0-linux64.tar.gz \
+	&& wget -qO- -O geckodriver.tar.gz https://github.com/mozilla/geckodriver/releases/download/v0.29.0/geckodriver-v0.29.0-linux64.tar.gz \
 	&& tar -xvzf geckodriver.tar.gz \
 	&& chmod +x geckodriver \
 	&& ln -s /opt/geckodriver /usr/bin/geckodriver \
 	&& export PATH=$PATH:/usr/bin/geckodriver
 
-ADD . /zap/
+ADD *.py /zap/

```

Comment:
One ADD command is better (each command adds a layer): https://docs.docker.com/develop/develop-images/dockerfile_best-practices/. Also, with this PR we are going to use .dockerignore so all other files are already ignored if we do ADD . /zap/

Reply:
It's true that with the docker ignore file, this can be done with one copy command. The early version of Podman did not use this ignore file. The newer version uses this file also.